srv/modules/runners/cmd.py

```diff
@@ -0,0 +1,13 @@
+import salt.client
+
+from subprocess import check_output
+
+def run(**kwargs):
```

Comment:
I couldn't find any place where you actually call this, but in the light of the recent changes with `__utils__`[0] and the issues with `__pycache__` and runner importing, I suggest to move this to the `__utils__` directory.
.. Unless it's purpose is solely to run arbitrary commands on the master from the commandline.


[0] https://github.com/SUSE/DeepSea/pull/1116

Reply:
It's called right here https://github.com/SUSE/DeepSea/pull/1117/files#diff-4286400fafff454ffeb90b4624230d9aR1

Reply:
But its certainly meant to be a utility function...so :+1:

Reply:
> It's called right here https://github.com/SUSE/DeepSea/pull/1117/files#diff-4286400fafff454ffeb90b4624230d9aR1

woops, my bad.